applications/openshift/kubelet/kubelet_enable_protect_kernel_defaults/rule.yml

```diff
@@ -5,7 +5,53 @@ prodtype: ocp4
 title: 'kubelet - Enable Protect Kernel Defaults'
 
 description: |-
+  <p>
   Protect tuned kernel parameters from being overwritten by the kubelet.
+  </p>
+
+  <p>
+  Before enabling this kernel parameter, it's important and
+  necessary to first create a <tt>MachineConfig</tt> object that persist
+  the required sysctl's. The manifest may look as follows:
+  </p>
+
+  <pre>
+  ---
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: master
+    name: 75-master-kubelet-sysctls
+  spec:
+    config:
+      ignition:
+        version: 3.1.0
+      storage:
+        files:
+        - contents:
+            source: data:,vm.overcommit_memory%3D1%0Avm.panic_on_oom%3D0%0Akernel.panic%3D10%0Akernel.panic_on_oops%3D1%0Akernel.keys.root_maxkeys%3D1000000%0Akernel.keys.root_maxbytes%3D25000000%0A
+          mode: 0644
+          path: /etc/sysctl.d/90-kubelet.conf
+          overwrite: true
+  </pre>
+  
+  <p>
+  This will need to be done for each relevant <tt>MachineConfigPool</tt>
+  in the cluster.
+  </p>
+
+  <p>
+  After enabling this and after the changes have successfully rolled out
+  to the whole cluster, it will now be possible to set the
+  <tt>protectKernelDefaults</tt> parameter. 
+  </p>
+
+  <p>
+  To configure, follow the directions in
+  {{{ weblink(link="https://docs.openshift.com/container-platform/4.5/nodes/nodes/nodes-nodes-garbage-collection.html#nodes-nodes-garbage-collection-configuring_nodes-nodes-configuring",
```

Comment:
Wrong link?

Reply:
of course I added the wrong link! lol
Let me fix it.